Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Base stage for building final images
+FROM phusion/passenger-ruby27:2.0.0 as BASE_IMAGE
```

Comment:
I think Tom was having issues with the capitalized BASE_IMAGE/DEVELOPMENT_IMAGE names. Could you lowercase them here and we can make the same changes on Hyku Addons :)

(Dont forget to change the image name in the docker-compose file too)